src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -227,8 +227,8 @@ export class AttachedAccountsTreeItem implements IAzureParentTreeItem {
                     // Default to Mongo if the value is a string for the sake of backwards compatiblity
                     // (Mongo was originally the only account type that could be attached)
```

Comment:
Your branches aren't correct. It should something like "ps/fixRangeError" merging into "rel/0.7.1"

"rel/0.7.1" should be a clean branch off of "v0.7.0"

Reply:
Also you might as well do the changelog/version bump in this PR